src/test/java/api/items/ItemApiExamples.java

```diff
@@ -1383,6 +1429,145 @@ public void cannotRemoveHRID() throws Exception {
+      "Unrecognized name"
+    ));
+  }
+
+  @Test
+  public void cannotUpdateItemWithUnrecognisedStatusName() throws Exception {
+    JsonObject createdInstance = createInstance(smallAngryPlanet(UUID.randomUUID()));
+
+    UUID holdingId = holdingsStorageClient.create(
+      new HoldingRequestBuilder()
+        .forInstance(UUID.fromString(createdInstance.getString("id"))))
+      .getId();
+
+    IndividualResource postResponse = itemsClient.create(new ItemRequestBuilder()
+      .forHolding(holdingId)
+      .temporarilyInReadingRoom());
+
+    JsonObject createdItem = postResponse.getJson();
+    assertThat(createdItem.getJsonObject("status").getString("name"),
+      is(ItemStatusName.AVAILABLE.value()));
+
+    JsonObject updatedItem = createdItem.copy()
+      .put("status", new JsonObject().put("name", "Unrecognized name"));
+
+    Response updateResponse = updateItem(updatedItem);
+    assertThat(updateResponse, hasValidationError(
+      "Undefined status specified",
+      "status.name",
+      "Unrecognized name"
+    ));
+  }
+
+  @Test
+  public void cannotRemoveStatusFromItem() throws Exception {
+    JsonObject createdInstance = createInstance(smallAngryPlanet(UUID.randomUUID()));
+
+    UUID holdingId = holdingsStorageClient.create(
+      new HoldingRequestBuilder()
+        .forInstance(UUID.fromString(createdInstance.getString("id"))))
+      .getId();
+
+    IndividualResource postResponse = itemsClient.create(new ItemRequestBuilder()
+      .forHolding(holdingId)
+      .temporarilyInReadingRoom());
+
+    JsonObject createdItem = postResponse.getJson();
+    assertThat(createdItem.getJsonObject("status").getString("name"),
+      is(ItemStatusName.AVAILABLE.value()));
+
+    JsonObject updatedItem = createdItem.copy();
+    updatedItem.remove("status");
+
+    Response updateResponse = updateItem(updatedItem);
+    assertThat(updateResponse,
+      hasValidationError("Status is a required field", "status", null)
+    );
+  }
+
+  @Test
+  public void cannotRemoveStatusNameFromItem() throws Exception {
+    JsonObject createdInstance = createInstance(smallAngryPlanet(UUID.randomUUID()));
+
+    UUID holdingId = holdingsStorageClient.create(
+      new HoldingRequestBuilder()
+        .forInstance(UUID.fromString(createdInstance.getString("id"))))
+      .getId();
+
+    IndividualResource postResponse = itemsClient.create(new ItemRequestBuilder()
+      .forHolding(holdingId)
+      .temporarilyInReadingRoom());
+
+    JsonObject createdItem = postResponse.getJson();
+    assertThat(createdItem.getJsonObject("status").getString("name"),
+      is(ItemStatusName.AVAILABLE.value()));
+
+    JsonObject updatedItem = createdItem.copy()
+      .put("status", new JsonObject());
+
+    Response updateResponse = updateItem(updatedItem);
+    assertThat(updateResponse,
+      hasValidationError("Status is a required field", "status", null)
+    );
+  }
+
+  @Test
+  public void canHandleStatusDateProperty() throws Exception {
```

Comment:
What does handle mean in this situation?

Reply:
I've renamed it to `statusDatePropertyPresentOnStatusUpdated` does it work better?

Reply:
I was trying to better understand. What does the test demonstrate, that after an item status is changed that the `status changed date` is present in the response?

Reply:
The test demonstrates that the status.date is forwarded from item-storage and has expected format and is within 2 seconds from _now()_.